cpp/src/arrow/dataset/file_rados_parquet.h

```diff
@@ -54,6 +54,12 @@
 namespace arrow {
 namespace dataset {
 
+/// \addtogroup dataset-file-formats
+///
+/// @{
+
+/// \brief A class that specifies the connection and disconnection from the
```

Comment:
We can say "An interface to connect to a RADOS cluster and hold the connection information for usage in later stages."

Reply:
Done.